set.mm

```diff
@@ -611012,6 +611013,84 @@ fixed reference functional determined by this vector (corresponding to
 $( (End of Norm Megill's mathbox.) $)
 
 
+$( Begin $[ set-mbox-sn.mm $] $)
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Steven Nguyen
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Utility theorems
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    ioin9i8.1 $e |- ( ph -> ( ps \/ ch ) ) $.
+    ioin9i8.2 $e |- ( ch -> -. th ) $.
+    ioin9i8.3 $e |- ( ps -> th ) $.
+    $( Miscellaneous inference creating a biconditional from an implied
+       converse implication.  (Contributed by Steven Nguyen, 17-Jul-2022.) $)
+    ioin9i8 $p |- ( ph -> ( ps <-> th ) ) $=
+      ( wn ord syl6 con4d impbid2 ) ABDGABDABHCDHABCEIFJKL $.
+  $}
+
+  ${
+    jaodd.1 $e |- ( ph -> ( ps -> ( ch -> th ) ) ) $.
+    jaodd.2 $e |- ( ph -> ( ps -> ( ta -> th ) ) ) $.
+    $( Double deduction form of ~ jaoi .  (Contributed by Steven Nguyen,
+       17-Jul-2022.) $)
+    jaodd $p |- ( ph -> ( ps -> ( ( ch \/ ta ) -> th ) ) ) $=
+      ( wi wo jao syl6c ) ABCDHEDHCEIDHFGCDEJK $.
+  $}
+
+  $( The proper subset of a set is also a set.  (Contributed by Steven Nguyen,
+     17-Jul-2022.) $)
+  pssexg $p |- ( ( A C. B /\ B e. C ) -> A e. _V ) $=
+    ( wpss wss wcel cvv pssss ssexg sylan ) ABDABEBCFAGFABHABCIJ $.
+
+  $( A proper superset is nonempty.  (Contributed by Steven Nguyen,
+     17-Jul-2022.) $)
+  pssn0 $p |- ( A C. B -> B =/= (/) ) $=
+    ( wpss c0 wceq npss0 psseq2 mtbiri necon2ai ) ABCZBDBDEJADCAFBDAGHI $.
+
+  $( Classes are proper subclasses if and only if their power classes are
+     proper subclasses.  (Contributed by Steven Nguyen, 17-Jul-2022.) $)
+  psspwb $p |- ( A C. B <-> ~P A C. ~P B ) $=
+    ( wss wne wa cpw wpss sspwb pweqb necon3bii anbi12i df-pss 3bitr4i ) ABCZAB
+    DZEAFZBFZCZPQDZEABGPQGNROSABHABPQABIJKABLPQLM $.
+
+  $( Proper subset theorem for Cartesian product.  (Contributed by Steven
+     Nguyen, 17-Jul-2022.) $)
+  xppss12 $p |- ( ( A C. B /\ C C. D ) -> ( A X. C ) C. ( B X. D ) ) $=
+    ( wpss wa cxp wss wne pssss xpss12 syl2an wn simpl pssne necomd neneq pssn0
+    wceq c0 intnanrd 3syl wb xp11 mtbird neqne syl df-pss sylanbrc ) ABEZCDEZFZ
+    ACGZBDGZHZUMUNIZUMUNEUJABHCDHUOUKABJCDJABCDKLULUNUMSZMZUPULUQBASZDCSZFZULUJ
+    BAIZVAMUJUKNUJABABOPVBUSUTBAQUAUBUJBTIDTIUQVAUCUKABRCDRBDACUDLUEURUNUMUNUMU
+    FPUGUMUNUHUI $.
+
+  ${
+    elpw2i.1 $e |- B e. _V $.
+    $( Membership in a power set, biimplication.  (Contributed by Steven
+       Nguyen, 17-Jul-2022.) $)
+    elpw2i $p |- ( A C_ B <-> A e. ~P B ) $=
```

Comment:
I would call this theorem ~elpwbi ("b" for biconditional, "i" for "infernece because of the hypothesis).

Reply:
Yes. And "biconditional" instead of "biimplication" in the comment.